bindings/python/pinocchio/display/abstract_display.py

```diff
@@ -0,0 +1,70 @@
+from .. import libpinocchio_pywrap as pin
+from ..shortcuts import buildModelsFromUrdf, createDatas
+
+import time
+
+class AbstractDisplay(object):
```

Comment:
Maybe AbstractViewer is better here as naming.

Reply:
So, the reason why I chose the word "Display", and not "Viewer", is that these are not really the viewers, but just wrappers around them. A "Display" instance usually contains a variable called `viewer`, which represents the actual viewer doing the job of displaying the stuff. So I think calling the Pinocchio classes `<Something>Viewer` might be confusing. The word "Display" comes from a tiny class called `Display` which sometimes pops up in lots of Gepetto code, and which is basically a primitive version of `GepettoDisplay` (thus, it is now made redundant).
I don't care a lot about this, but I would rather spare myself the work of changing all the namings.

Reply:
I don't like the way of having two namings for the same functionality. In this case, the naming ViewerWrapper is better I would say.

Reply:
> I don't like the way of having two namings for the same functionality.

Why two namings? Classes are all consistently called `<Something>Display`. `GepettoDisplay` uses Gepetto Viewer, and `MeshcatDisplay` uses Meshcat Viewer. It seems perfectly consistent to me.

If you want ViewerWrapper, then I guess we should call the three classes `AbstractViewerWrapper`, `GepettoViewerWrapper` and `MeshcatViewerWrapper`? If you prefer... I think it's just a more complicated name and it will be a pain to change.

Reply:
Display seems not adapted, and we should focus on a single naming to name the same thing.
`MeshcatWrapper` is short and sufficient. `ViewerWrapper` for the parent class (we can forget the Abstract which is not needed). And GepettoViewer is the naming of the visualization tool, so we should go for `GepettoViewerWrapper`.

Reply:
Finally, I don't like calling it `Wrapper`. I'd rather stay with the current naming. I think it's clear enough, and I do not see an issue of duplicate naming: a Pinocchio display uses a viewer, there's a clear separation.

Reply:
See changes